### PR TITLE
feat: adapt platform danger zone section, extend explanation around p…

### PR DESCRIPTION
**Pull request by uleMeshcloud**
…ossible states

CU-861m6kd0x

### CODE REVIEW

File: docs/administration.platforms.md [hunk 1]
```diff
@@ -91,13 +91,15 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Platform Danger Zone
 
-Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+Via the "Danger Zone" tab you can deprecate, undo a deprecation or delete your meshPlatform.
 
-Deactivation will make the meshPlatform not selectable anymore for new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. In order to immediately stop metering as well, the Metering Configuration needs to be removed. All settings of the platform will remain in meshStack by default. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform's tenants when a meshPlatform is deactivated.
+**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel.
```
Comment (tfelix) [inline docs/administration.platforms.md:98]: d: maybe also write that an indicator will tell people that those platforms are deprecated visually.

**Review by tfelix — APPROVED** [review]

File: docs/administration.platforms.md [hunk 2]
```diff
@@ -91,13 +91,15 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Platform Danger Zone
```
Comment (sttomm) [inline docs/administration.platforms.md:94]: d: Perhaps better split it into 2 headlines. One for Deprecation and one for Deletion. I think that provides a better overview on what this section is about. Danger Zone leaves room for a lot of interpretation. But reading about deprecation and deletion you directly know what it is about.

File: docs/administration.platforms.md [hunk 3]
```diff
@@ -91,13 +91,15 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
 
-A reactivation of meshPlatforms is also possible. This will allow reaccessing and replicating non-deleted meshTenants of this meshPlatform again and make the meshPlatform available again for new meshTenants.
+**Reverting a deprecation** can be done by the `Reactivate meshPlatform` button. This allows your users to select the meshPlatform for new meshTenants again.
+
+**Deletion** will make the meshPlatform not selectable for new meshTenants and existing meshTenants not accessible via meshPanel anymore. No more replication will be executed. Metering instead will still be executed for non-deleted platform tenants of this meshPlatform for now. In order to immediately stop metering as well, the Metering Configuration needs to be removed. All settings of the platform will remain in meshStack by default. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform's tenants when a meshPlatform is deactivated.
```
Comment (sttomm) [inline docs/administration.platforms.md:102]: d: Do you want to merge this after your PR about actually doing the tenant deletion when a platform is deleted? If that's the case, which would make sense to me, you have to change the description of Deletion. We will actually delete all tenants right away and also remove the metering and replicator config. So the platform will be completely unaccessible via meshStack and not recoverable.

**Review by sttomm — CHANGES_REQUESTED** [review]

File: docs/administration.platforms.md [hunk 4]
```diff
@@ -91,13 +91,23 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
+2. Select the meshPlatform.
+3. Navigate to **Danger Zone**. Depending on the state of your meshPlatform. You can either see a `Deprecate meshPlatform` or `Reactivate meshPlatform` button.
+4. Click on the button.
 
-A reactivation of meshPlatforms is also possible. This will allow reaccessing and replicating non-deleted meshTenants of this meshPlatform again and make the meshPlatform available again for new meshTenants.
+### Delete Platform
+
+**Deletion** will make the meshPlatform not selectable for new meshTenants and existing meshTenants not accessible via meshPanel anymore. No more replication will be executed. Once you delete a meshPlatform, meshStack will automatically delete all meshTenants of the meshPlatform without the deleting the platform tenants in the cloud platform. The deletion process of a meshPlatform will also remove the replicator and metering configurations. Therefore, your meshPlatform will be completely inaccessible via meshStack and can not be recovered. As meshStack does not delete or do any updates on platform's tenants itself when a meshPlatform is deleted, direct access in the cloud platform to the platform tenants is still possible.
```
Comment (sttomm) [inline docs/administration.platforms.md:105]:
```suggestion
**Deletion** will make the meshPlatform not selectable for new meshTenants and existing meshTenants not accessible via meshPanel anymore. No more replication will be executed. Once you delete a meshPlatform, meshStack will automatically delete all meshTenants of the meshPlatform without deleting the platform tenants in the cloud platform. The deletion process of a meshPlatform will also remove the replicator and metering configurations. Therefore, your meshPlatform will be completely inaccessible via meshStack and can not be recovered. As meshStack does not delete or do any updates on platform tenants themself when a meshPlatform is deleted, direct access in the cloud platform to the platform tenants is still possible.
```

Comment (uleMeshcloud) [reply docs/administration.platforms.md:105]: fixed to themselves

File: docs/administration.platforms.md [hunk 5]
```diff
@@ -91,13 +91,23 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Deprecate Platform
 
-Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel. Your users can recognize a meshTenant of a deprecated meshPlatform by a yellow bubble on the platform icon in the projects overview on the customer control plane. If your users are on the tenant control plane, a yellow status label will appear in the header of the control plane. You can also **undo the deprecation**. This allows your users to select the meshPlatform for new meshTenants again.
```
Comment (PolinaSadykova) [inline docs/administration.platforms.md:97]:
```suggestion
**Deprecation** will prevent the meshPlatform from being selected for new meshTenants. Existing meshTenants of this meshPlatform will still be replicated and can still be accessed via the meshPanel. Your users can recognize a meshTenant of a deprecated meshPlatform by a yellow bubble on the platform icon in the "Projects Overview" screen on the Customer Control plane. If the user is on the tenant control plane, a yellow status label will appear in the header of the control plane. You can also **undo the deprecation**. This allows your users to select the meshPlatform for new meshTenants again.

```

File: docs/administration.platforms.md [hunk 6]
```diff
@@ -91,13 +91,23 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
 
-Deactivation will make the meshPlatform not selectable anymore for new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. In order to immediately stop metering as well, the Metering Configuration needs to be removed. All settings of the platform will remain in meshStack by default. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform's tenants when a meshPlatform is deactivated.
```
Comment (PolinaSadykova) [inline docs/administration.platforms.md:98]: Will it be also on the screen or not?

File: docs/administration.platforms.md [hunk 7]
```diff
@@ -91,13 +91,23 @@ If you want to remove quotas from a platform just click the "-" button on the ri
...
+### Delete Platform
+
+**Deletion** will make the meshPlatform not selectable for new meshTenants and existing meshTenants not accessible via meshPanel anymore. No more replication will be executed. Once you delete a meshPlatform, meshStack will automatically delete all meshTenants of the meshPlatform without the deleting the platform tenants in the cloud platform. The deletion process of a meshPlatform will also remove the replicator and metering configurations. Therefore, your meshPlatform will be completely inaccessible via meshStack and can not be recovered. As meshStack does not delete or do any updates on platform tenants themselves when a meshPlatform is deleted, direct access in the cloud platform to the platform tenants is still possible.
+
+1. Navigate to **Platforms** > **meshPlatforms**.
+2. Select the meshPlatform.
+3. Navigate to **Danger Zone**.
+4. Click on `Delete meshPlatform` button and confirm the deletion with its identifier. A deletion **can not be reverted**.
```
Comment (PolinaSadykova) [inline docs/administration.platforms.md:110]:
```suggestion
### Delete Platform

**Deletion** will make the meshPlatform not selectable for new meshTenants and existing meshTenants not accessible via meshPanel anymore. No more replication will be executed. Once you delete a meshPlatform, meshStack will automatically delete all meshTenants of the meshPlatform without deleting the platform tenants in the cloud platform. The deletion process of a meshPlatform will also remove the replicator and metering configurations. Therefore, your meshPlatform will be completely inaccessible via meshStack and can not be recovered. As meshStack does not delete or do any updates on platform tenants themselves when a meshPlatform is deleted, direct access in the cloud platform to the platform tenants is still possible.

1. Navigate to **Platforms** > **meshPlatforms**.
2. Select the meshPlatform.
3. Navigate to **Danger Zone**.
4. Click on `Delete meshPlatform` button and confirm the deletion with its identifier. A deletion **can not be reverted**.
```

File: docs/administration.platforms.md (hunk 5 above)
Comment (PolinaSadykova) [reply docs/administration.platforms.md:97]: I could not find tenant control plane status ( e.g. depreciation) Could you help me with which screen it is?

**Review by PolinaSadykova — APPROVED** [review]

**Pull request merged**